src/extension.ts

```diff
@@ -105,26 +134,20 @@ async function findLookupPath(): Promise<string | undefined> {
       }
       return undefined;
     });
-  }
 
-  if (gitRoot) {
     return gitRoot;
-  } else {
-    return ws;
   }
+
+  return ws;
```

Comment:
It looks like this change completely disables the git root lookup. Is this intentional?

Reply:
@solaris765 Here I reduced the conditions, since the linter swore at a large number of conditions in the current function. gitRoot should remain working and changes should not affect this functionality. Correct me if you broke something